Include/internal/pycore_object.h

```diff
@@ -697,8 +697,52 @@ _PyObject_SetMaybeWeakref(PyObject *op)
     }
 }
 
+extern int _PyObject_ResurrectEndSlow(PyObject *op);
 #endif
 
+// Temporarily resurrects an object during deallocation. The refcount is set
+// to one.
+static inline void
+_PyObject_Resurrect(PyObject *op)
```

Comment:
How about naming it something like `_PyObject_ResurrectStart`? That would make its corresponding `_PyObject_ResurrectEnd` more obvious.

Reply:
I changed it to `_PyObject_ResurrectStart`